data_processing/pathology/refined_table/annotation/generate.py

```diff
@@ -0,0 +1,246 @@
+
+import click
+
+from data_processing.common.CodeTimer import CodeTimer
+from data_processing.common.config import ConfigSet
+from data_processing.common.custom_logger import init_logger
+from data_processing.common.sparksession import SparkConfig
+from data_processing.common.Neo4jConnection import Neo4jConnection
+import data_processing.common.constants as const
+from data_processing.pathology.common.build_geojson_from_bitmap import build_geojson_from_bitmap_pandas, concatenate_regional_geojsons_pandas
+from data_processing.pathology.common.utils import get_add_triple_str
+
+from pyspark.sql.functions import udf, lit, col, first, last, desc
+from pyspark.sql.window import Window
+import pandas as pd
+
+import yaml, os
+
+logger = init_logger()
+
+@click.command()
+@click.option('-t', '--template_file', default=None, type=click.Path(exists=True),
+              help="path to yaml template file containing information required for pathology proxy data ingestion. "
+                   "See data_ingestion_template.yaml.template")
+@click.option('-f', '--config_file', default='config.yaml', type=click.Path(exists=True),
+              help="path to config file containing application configuration. See config.yaml.template")
+@click.option('-l', '--label_file', default='data_processing/pathology/config/regional_etl_configuration.yaml',
+              type=click.Path(exists=True),
+              help="path to label configuration file containing application configuration.")
+@click.option('-p', '--process_string', default='geojson',
```

Comment:
- I really like the organization here, its very clean:)
- Was thinking that since the concat_geojson table is derived from the geojson table, I was thinking if the concat table is built, geojson should also be run?
- Maybe we could modify the process_string logic so that  if 'concat' is in the process_string, run geojson as well so concat_table gets updated with the most up-to-date geojson_table? and/or have `geojson,concat` as the default process_string so they both run together

- 
- I was maybe could we make the default process_string `geojson,concat` so that by default both tables are written?

Reply:
Great point! we can have `geojson,concat` as default, and still have the flexibility to run them separately if needed.